tests/booting/stub/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
+
+target_sources(app PRIVATE src/main.c)
```

Comment:
@nashif : If I recall correctly you said build scripts for this is not necessary.


I am not familiar with this stub. Do we need this build script or not?

Reply:
*shrug* I don't know what build script stands for in here -- we need to build the stub when we are in (eg) Arduino 101 and we want to run only on ARC; the stub runs on x86 and basically passes serial output around and does nothing else / viceversa with x86|arc. 
When I just dropped in to build, it failed because it coudn't find cmake build info, so I copied Hello World's

Reply:
build script - Makefile/CMakeLists.txt/SConscript etc.


Not sure what you are saying, but sounds like this is OK then :)

Reply:
should be ok, not sure in what context I said it is not needed, this is a test stub for multi-core systems, so it was not a test on its own.